src/System.Management.Automation/engine/CommandCompletion/CompletionAnalysis.cs

```diff
@@ -1756,13 +1756,15 @@ private List<CompletionResult> GetResultForAttributeArgument(CompletionContext c
             {
                 PropertyInfo[] propertyInfos = attributeType.GetProperties(BindingFlags.Public | BindingFlags.Instance);
                 List<CompletionResult> result = new List<CompletionResult>();
-                foreach (PropertyInfo pro in propertyInfos)
+                foreach (PropertyInfo property in propertyInfos)
                 {
-                    //Ignore TypeId (all attributes inherit it)
-                    if (pro.Name != "TypeId" && (pro.Name.StartsWith(argName, StringComparison.OrdinalIgnoreCase)))
+                    // Ignore getter-only properties, including 'TypeId' (all attributes inherit it).
+                    if (!property.CanWrite) { continue; }
```

Comment:
We use `if () { // simple statement }` all over the code base. But I don't think we should allow the first pattern in your comment. The if body should always be in `{ }`.

Reply:
Certainly agree that we should always use braces.

In a world where we no longer use teletypes (i.e. where screens are big and newlines are cheap), I tend to feel that we should always just hang the line so that as my eyes scan down a document at an approximate indentation level and always know what's going on. **But**, that's just a style preference, and inline-braces are still pretty good to me.